<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionCerrar"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionAbrir"/>
   <addaction name="actionGuardar"/>
   <addaction name="actionBrowser"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/open.png</normaloff>:/resources/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/save.png</normaloff>:/resources/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionVer_mapa">
   <property name="text">
    <string>Ver mapa</string>
   </property>
  </action>
  <action name="actionOcultar_mapa">
   <property name="text">
    <string>Ocultar mapa</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/new.png</normaloff>:/resources/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionBrowser">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/web.png</normaloff>:/resources/icons/web.png</iconset>
   </property>
   <property name="text">
    <string>Browser</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
